cmd/juju/action/status.go

```diff
@@ -138,6 +140,13 @@ func resultToMap(result params.ActionResult) map[string]interface{} {
 
 	}
 	item["status"] = result.Status
+
+	if result.Completed.Equal(time.Time{}) {
```

Comment:
`result.Completed` uses the Zero valued time (epoch) to indicate that the field has not been completed.